test/hooks/test-trace-hapi.js

```diff
@@ -122,6 +123,9 @@ Object.keys(versions).forEach(function(version) {
     });
 
     it('should accurately measure get time, after + get', function(done) {
+      if (version.substring(4) > 10) {
```

Comment:
https://github.com/hapijs/hapi/issues/2850 The `after` method was removed in 11.